test/SlowTests/Server/Replication/PullReplicationTests.cs

```diff
@@ -309,7 +309,9 @@ public async Task DisablePullReplicationOnHub()
 
                 pullDefinition.Disabled = true;
                 pullDefinition.TaskId = saveResult.TaskId;
-                await hub.Maintenance.ForDatabase(hub.Database).SendAsync(new PutPullReplicationAsHubOperation(pullDefinition));
+                var updateResult = await hub.Maintenance.ForDatabase(hub.Database).SendAsync(new PutPullReplicationAsHubOperation(pullDefinition));
+
+                await Server.ServerStore.Cluster.WaitForIndexNotification(updateResult.RaftCommandIndex);
```

Comment:
aren't we doing that on the server itself?

Reply:
you're right, we already do that in `DatabaseConfigurations`
I guess we need to wait for replication loader to `HandleDatabaseRecordChange` then 
I'll try to do that